src/Components/Activities/ActivitiesPagination/ActivitiesPagination.js

```diff
@@ -0,0 +1,56 @@
+import {useState, useEffect} from "react"
+import ArrowLeft from '@material-ui/icons/ArrowLeft';
+import ArrowRight from '@material-ui/icons/ArrowRight';
+import ActivitiesCards from '../ActivitiesCards/ActivitiesCards';
+import {getAllActivities} from '../../../Services/activityService';
+const cards = 9;
+
+const ActivitiesPagination = () => {
```

Comment:
you can make a Generic Pagination component passing by attribute the items list and then you can have a list of currentItems instead of just items and also the  <ActivitiesCards activities={items} /> can be send by attribute on a function :
```
const ListPagination = ({items, showItemsListComponent}) => {
    const [currentItems, setCurrentItems]=useState(null);
    const [currentPag, setCurrentPag]= useState(0);
    
   useEffect(()=>{
        if(items.length>1)setCurrentItems([...items].splice(0,cards))
    },[items])

....

return (
        <div className='paginacion'>
            <div className='paginacion_activities'>
                {showItemsListComponent(currentItems)}
            </div>
          <div className='paginacion_buttons'>
                <button className='paginacion_btn' onClick={()=>prev()}><ArrowLeft/></button>
                <button className='paginacion_btn' onClick={()=>next()}><ArrowRight/></button>
          </div>
        </div>
    )
}
```